test/basic.js

```diff
@@ -60,6 +60,29 @@ describe('Basic Tests', () => {
       .expect(404, done);
   });
 
+  // GET /training
+  it('GET /training', (done) => {
+    request.agent(app)
+      .get('/training')
+      .expect(302, done);
```

Comment:
The site redirects these to the homepage. I was surprised by this and I'm pretty sure @kurtlewis had it this way from the start. If you try to go to ucceasambassadors.com/training without first logging, you'll get an error notification on the main page saying you have to be logged in first. 

If this is the way to keep it, should I be testing that it redirects to the homepage or is this adequate?

Reply:
Oops, I didn't mean to say that it goes to a special 401 page, but that when you go to the training page (if you're not logged in), it redirects to the homepage and sets the HTTP response to 401. I believe if you try editing your test with my example, it should work since when you open the networks tab, you can see that request does indeed result in a 401.

Reply:
+1 to testing like in Mihir's snippet. The correct behavior is that it goes to the main page with the 401 error code.

I made it go to the home page with an error message because I thought it was a better user experience than having a dedicated error page.

Reply:
I resolved this issue in my last commit.